app/models/__init__.py

```diff
@@ -63,7 +63,7 @@ def team(self) -> Optional[Team]:
         return self._teams[0] if self._teams else None
 
     def __repr__(self):
-        return f"<User(id={self.id}, name={self.name}, email={self.email}, team_id={self.team_id})>"
+        return f"<User(id={self.id}, name={self.name}, email={self.email}, team_id={self.team})>"
```

Comment:
Do we need to print out the team? My concern is the recursive structure that would come out (team contains users, users contain team, etc.)